src/status_im/contexts/wallet/common/utils/wallet_connect.cljs

```diff
@@ -0,0 +1,21 @@
+(ns status-im.contexts.wallet.common.utils.wallet-connect
```

Comment:
What about having the wallet-connect stuff in its own namespace e.g. `status-im.contexts.wallet.wallet-connect`?

Reply:
I agree. 👍

Reply:
Moved it 👍